birdhouse/components/weaver/post-docker-compose-up

```diff
@@ -92,6 +92,16 @@ fi
 echo "  Requested Weaver WPS providers: [${WEAVER_WPS_PROVIDERS}]"
 echo "  Will retry requests at most for ${WEAVER_WPS_PROVIDERS_MAX_TIME}s"
 
+if [ -z "$WEAVER_CURL_IMAGE" ]; then
+    WEAVER_CURL_IMAGE="curlimages/curl:7.87.0"
+fi
+
+# To know when a docker run was started in case it hangs.
+DOCKER_RUN_TAG="`date -Isecond | sed 's/:/_/g' | sed 's/+/p/g'`_$RANDOM"
+curl_cmd() {
+    docker run --rm --name weaver_post_curl_$DOCKER_RUN_TAG $WEAVER_CURL_IMAGE "$@"
+}
```

Comment:
Would it be possible to make undefined `WEAVER_CURL_IMAGE` act as the previous method, using directly `curl`? 
This way, servers that were working without it would not change behavior and accumulate extra images.
A server that needs this specific support would have to provide `WEAVER_CURL_IMAGE` explicitly (explicit empty could still be using the default defined here).

Reply:
```
$ docker images |grep curl
curlimages/curl                     7.87.0                                           3148ec916ea7   2 days ago      15MB
```
I don't think the effort is worth saving 15MB on disk.  I'd rather keep things simple than doing premature optimization, especially not for 15MB.

Reply:
Ok. Fair enough.